lib/rules/svg-namespace-matches.js

```diff
@@ -10,7 +10,8 @@ function svgNamespaceMatches(node, virtualNode) {
 
     // element is svg namespace if its parent is an svg element
     return !!closest(virtualNode, 'svg');
-  } catch {
+  } catch (e) {
+    console.debug(e);
```

Comment:
Can't have this. Axe-core can't just put things into the console as the console may be piped out to a place that isn't expecting us to inject things into it.

```suggestion
  } catch {
```